kotest-assertions/kotest-assertions-core/src/jvmMain/kotlin/io/kotest/matchers/equality/reflection.kt

```diff
@@ -299,42 +299,6 @@ fun <T : Any> beEqualToIgnoringFields(
    }
 }
 
-/**
- * Matcher that compares values without using field by field comparison.
- *
- * This matcher should be used to check equality of two class for which you want to consider their fields for equality
- * instead of its `equals` method.
- *
- * This matcher recursively check equality of given values till we get a java class, kotlin class or fields for which we have
- * specified to use default shouldBe. Once we get a java class, kotlin class or specified field the equality of that fields
- * will be same as that we get with shouldBe matcher.
- *
- * @param other the other class to which equality need to be checked.
- * @param ignorePrivateFields specify whether private fields should be ignored in equality check or not, default value is true
- * @param ignoreComputedFields specify whether computed fields should be ignored in equality check or not, default value is true
- * @param useDefaultShouldBeForFields fully qualified names of data type for which we need to use default shouldBe, default empty list.
- *
- * */
-@Deprecated(
-   message = "This will be removed in Kotest 7.0",
```

Comment:
Nit:
I mean, we did promise to keep it until Kotest 7.0, not 6.0

Reply:
(applies to most of the removed assertions)

Reply:
But that was a typo I think becuse they were deprecated in 2022